src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -955,10 +955,19 @@ LottieMarker* LottieParser::parseMarker()
     auto marker = new LottieMarker;
     
     while (auto key = nextObjectKey()) {
-        if (KEY_AS("cm")) marker->name = getStringCopy();
-        else if (KEY_AS("tm")) marker->time = getFloat();
-        else if (KEY_AS("dr")) marker->duration = getFloat();
-        else skip(key);
+        if (KEY_AS("cm")) {
+            if (peekType() == kObjectType) {
```

Comment:
If I remember correctly, the "cm" attribute is written in JSON string not object. We should parse them for string type.

Reply:
```js
"markers": [
    { "tm": 0, "cm": "{\\"name\\":\\"bird\\"}", "dr": 22 },
    { "tm": 22, "cm": "{\\"name\\":\\"explosion\\"}", "dr": 11 },
    { "tm": 33, "cm": "{\\"name\\":\\"feather\\"}", "dr": 77 }
]
```

Reply:
Hmm... It's an object. Any side issues?

Reply:
I mean, since the "cm" is string type even if it's actually JSON format, we may need to parse stringified JSON to object then parse again in object type.

![Screenshot 2024-04-22 at 10 19 50 AM](https://github.com/thorvg/thorvg/assets/11167117/a8ca0064-de4a-43c0-aa03-d4006c120009)

I tried an example with JSON "cm". It fails.

![Screenshot 2024-04-22 at 10 22 18 AM](https://github.com/thorvg/thorvg/assets/11167117/eced3755-072f-46dc-8f01-c204d3148fd8)
[marker_sample.json](https://github.com/thorvg/thorvg/files/15056043/marker_sample.json)

Reply:
@tinyjin Sorry, I don't see any issue here. It handles the CM name as a string. Could you please suggest the correct logic you think? I also can't see any errors with your sample file.

Reply:
OK, thanks for check. I will soon test it again.